vendor/wax.cartodb.js

```diff
@@ -3329,23 +3329,58 @@ wax.g.connector = function(options) {
 // Get a tile element from a coordinate, zoom level, and an ownerDocument.
 wax.g.connector.prototype.getTile = function(coord, zoom, ownerDocument) {
     var key = zoom + '/' + coord.x + '/' + coord.y;
+    var img = null;
     if (!this.cache[key]) {
-        var img = this.cache[key] = new Image(256, 256);
-        this.cache[key].src = this.getTileUrl(coord, zoom);
-        this.cache[key].setAttribute('gTileKey', key);
-        this.cache[key].setAttribute("style","opacity: "+this.opacity+"; filter: alpha(opacity="+(this.opacity*100)+");");
-        this.cache[key].onerror = function() { img.style.display = 'none'; };
+      this.cache[key] = new Image(256, 256);
+      img = this.cache[key];
```

Comment:
As we have the reference inside the `img` variable maybe we could replace the next `this.cache[key]` ocurrences with it

Reply:
I prefer being explicit on things added to the cache but I get your point.
`img` is only used inside here for the `onerror` callback. If it wasn't for it I wouldn't create `var img` at all.